client/src/Components/MessageWindow/MessageBox.jsx

```diff
@@ -57,28 +66,19 @@ export default function MessageBox(props) {
 
   return (
 
-      <div className ={`msg_box${isSender? ' sent':''}`}>
-        <div>
-          <div className={`line${isSender? ' sent':''}`}></div>
-          <div className = 'username'>{isSender? 'me': props.sender}</div>
-          <div className = 'content'>
-          {url !== '' ? (
-            img === false ? (
-              <button onClick={() => showSnap()}>show snap</button>
-            ) : (
-              <div>
-                <img src={url.url} />
-                {showSaveButton && (
-                  <button onClick={() => handleSave()}>Save Image</button>
-                )}
-              </div>
-            )
-          ) : (
-            content
-          )}
-          </div>
-        </div>
-      </div>
+    <div className ={`msg_box${isSender? ' sent':''}`}>
+      <div>
+      <div className={`line${isSender? ' sent':''}`}></div>
+      <div className = 'username'>{isSender? 'me': props.sender}</div>
+      {/* <div className = 'content'>{props.content}</div> */}
+      <div className = 'content'>{
+         url!==''? (img === false ?
+         <button onClick={()=>{showSnap()}} >
+         show snap
+       </button> : <div><button onClick={(e)=>{download()}}>download</button><img src= {url.url}/></div>) : content}
+       </div>
+       </div>
+
```

Comment:
There appears to be a missing div container here, but also server-dev will not connect with this branch